### DISCUSSION TITLE
New Components

### DISCUSSION THREAD

**Discussion by ieedan** [ieedan/geist-ui-svelte#69] [Ideas]
Put your new component requests here and we can talk about how they look and expected behavior.

**Comment by SikandarJODD** [comment]
## Components i felt should be added : 
1. ButtonGroup : 
![image](https://github.com/ieedan/geist-ui-svelte/assets/93428946/78a66d75-ca14-4701-bd46-3a8cb7afa590)

2. Divider Width Should be Flexible, we can add <code>width</code>  attribute : https://geist-ui-svelte.dev/components/divider 
3.  we should add more examples on Drawer : https://geist-ui-svelte.dev/components/drawer 
4. Label Component ?

**Reply by ieedan** [reply]
Feel free to open a PR for any of these. And any of the previous you mentioned. Make sure your code is Tailwind 3.4.1 compatible.

**Reply by ieedan** [reply]
Please split them up into individual issues first and we can take them one at a time.

**Reply by ieedan** [reply]
The button group component was added
